dash-renderer/src/reducers/error.js

```diff
@@ -8,6 +8,11 @@ const initialError = {
 export default function error(state = initialError, action) {
     switch (action.type) {
         case 'ON_ERROR': {
+            // log errors to the console for stack tracing and so they're
+            // available even with debugging off
+            /* eslint-disable-next-line no-console */
+            console.error(action.payload.error);
```

Comment:
re: @chriddyp 's note:
> Also, side note: the actual JavaScript exception for this error isn't in the console so I'm not able to dig into the unminified code. Seems like devtools is catching the error but not re-raising it or printing it. I think that it used to send it to the console (hence the "open your browser's console" message), but I'm not sure.

Not clear to me when that logging disappeared but yes it's certainly useful. Any objection to logging it from the reducer here? A bit odd to have a reducer make side-effects, but this seems like the DRYest way to handle it, and it'll work whether debug mode is on or off.